### PR TITLE
Clarify README about supported versions (post v7 release)

### PR DESCRIPTION
**Pull request by akx**
The readme implies v7 is yet to be released, though it's been out for a while now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +32,7 @@ To learn more about Notebook v7: https://jupyter.org/enhancement-proposals/79-no
 
 ### Classic Notebook v6
 
-Maintenance and security-related issues are now being addressed in the [`6.4.x`](https://github.com/jupyter/notebook/tree/6.4.x) branch.
+Maintenance and security-related issues [only](https://github.com/jupyter/notebook-team-compass/issues/5#issuecomment-1085254000) are now being addressed in the [`6.4.x`](https://github.com/jupyter/notebook/tree/6.4.x) branch.
```
Comment (jtpio) [inline README.md:35]: Maybe we can even drop the mention of `6.4.x`, and just mention `6.5.x`?

Comment (akx) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -33,7 +32,7 @@ To learn more about Notebook v7: https://jupyter.org/enhancement-proposals/79-no
 
 ### Classic Notebook v6
 
-Maintenance and security-related issues are now being addressed in the [`6.4.x`](https://github.com/jupyter/notebook/tree/6.4.x) branch.
+Maintenance and security-related issues [only](https://github.com/jupyter/notebook-team-compass/issues/5#issuecomment-1085254000) are now being addressed in the [`6.5.x`](https://github.com/jupyter/notebook/tree/6.5.x) branch.
 
 A `6.5.x` branch will be soon created and will depend on [`nbclassic`](https://github.com/jupyter/nbclassic) for the HTML/JavaScript/CSS assets.
```
Comment (jtpio) [inline README.md:37]: And we can probably remove this line as the `6.5.x` branch now exists.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**